pkg/util/types/patch.go

```diff
@@ -70,3 +71,8 @@ func UnmarshalPatch(patch []byte) ([]PatchOperation, error) {
 
 	return p, err
 }
+
+func EscapeJSONPointer(ptr string) string {
```

Comment:
Side note: Not sure why this package is called types.

Reply:
I plan to move all patch related stuff to a new package called `apimachinery/patch`, that should solve this naming mess.